Backend/app/rider-platform/rider-app/Main/src/Tools/Notifications.hs

```diff
@@ -572,6 +577,29 @@ notifyOnNewMessage booking message = do
       title = T.pack "Driver"
       body =
         unwords
-          [ message
+          [ decodedMessage
           ]
   notifyPerson person.merchantId notificationData
+
+decodeLanguage :: Text -> Maybe Maps.Language -> Text
+decodeLanguage msg language =
+  case Map.lookup msg dataMap of
+    Just messages -> do
+      let message = getField (Just Maps.ENGLISH) messages ""
+      if isNothing language
+        then message
+        else getField language messages message
+    Nothing -> msg
+
+getField :: Maybe Maps.Language -> DChatSuggestions.Messages -> Text -> Text
+getField language DChatSuggestions.Messages {..} message = case language of
+  Just lan ->
+    case lan of
+      Maps.ENGLISH -> en_us
+      Maps.HINDI -> hi_in
+      Maps.KANNADA -> kn_in
+      Maps.TAMIL -> ta_in
+      Maps.MALAYALAM -> ml_in
+      Maps.BENGALI -> bn_in
+      Maps.FRENCH -> message
+  Nothing -> message
```

Comment:
can be refactored in more straightforward way:

```
decodeLanguage :: Text -> Maybe Maps.Language -> Text
decodeLanguage msg mbLanguage =
  case Map.lookup msg dataMap of
    Just messages -> do
      let language = fromMaybe Maps.ENGLISH mbLanguage
      getField messages language
    Nothing -> msg

getField :: DChatSuggestions.Messages -> Maps.Language -> Text
getField DChatSuggestions.Messages {..} = \case
  Maps.ENGLISH -> en_us
  Maps.HINDI -> hi_in
  Maps.KANNADA -> kn_in
  Maps.TAMIL -> ta_in
  Maps.MALAYALAM -> ml_in
  Maps.BENGALI -> bn_in
  Maps.FRENCH -> en_us -- default english
```

Reply:
@roman-bodavskiy 

can we do this?

decodeLanguage :: Text -> Maybe Maps.Language -> Text
decodeLanguage msg mbLanguage =
  let language = fromMaybe Maps.ENGLISH mbLanguage
  in Map.findWithDefault msg language dataMap

getField :: DChatSuggestions.Messages -> Maps.Language -> Text
getField DChatSuggestions.Messages {..} language = Map.findWithDefault en_us language
  (Map.fromList
    [ (Maps.ENGLISH, en_us)
    , (Maps.HINDI, hi_in)
    , (Maps.KANNADA, kn_in)
    , (Maps.TAMIL, ta_in)
    , (Maps.MALAYALAM, ml_in)
    , (Maps.BENGALI, bn_in)
    , (Maps.FRENCH, en_us) -- default English
    ])

Reply:
yes we can, but solution with `\case` I like more, because when we add new language, compiler will fail and we will have to add new `case`, so code will be always actual. That's why I like using `case` in code